src/logistration/LoginHelpLinks.jsx

```diff
@@ -61,6 +75,9 @@ const LoginHelpLinks = (props) => {
           default: <></>,
         }}
       />
+      <button type="button" className="mt-1 field-link position-relative" style={{ top: '-10px' }}>
```

Comment:
Remove this unnecessary button element around the link and use `mt-0` class instead of setting inline styles.

Reply:
We also need to add this in `LoginPage.jsx` instead of this component otherwise, it will add this link on the forgot password page as well.